lib/components/text/serif.js

```diff
@@ -21,7 +21,7 @@ Serif.propTypes = {
 
 const styles = StyleSheet.create({
   default: {
-    fontSize: 16
+    fontSize: 17
```

Comment:
I eyeballed the spec and Katarina's issue, and I think this subtle change looks more like what it's supposed to.